docs/tasks/administer-cluster/coredns.md

```diff
@@ -0,0 +1,41 @@
+---
+approvers:
+- johnbelamaric
+title: Using CoreDNS for Service Discovery
+---
+
+{% include feature-state-alpha.md %}
+
+{% capture overview %}
+This page describes how to enable CoreDNS instead of kube-dns for service
+discovery.
+{% endcapture %}
+
+{% capture prerequisites %}
+* Kubernetes version 1.9 and above.
```

Comment:
Replace this line with:
```
{% include task-tutorial-prereqs.md %}
```
The `min-kubernetes-server-version` declaration in the title block will populate the minimum version required. 👍